README.md

```diff
@@ -8,6 +8,24 @@ This is not an official Google product.
 
 Author: David Tomaschik <dwt@google.com>
 
+
+### Installation using Docker###
+1. Navigate to the folder where the Dockerfile is located.
+
+2. Run the command below to build a docker image for the scoreboard and tag it as "scoreboard". <br/>
+  docker build -t "scoreboard" .
```

Comment:
Indenting commands by 4 spaces should result in rendering as pre-formatted text.